### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by Harryprasanth** [pnp/powershell#3922] [Q&A]
In Powershell7 IDE, Running Script **Get-PnpPowerPlatformEnvironment** getting the error showing that 

Get-PnpPowerPlatformEnvironment: The 'Get-PnpPowerPlatformEnvironment' command was found in the module 'PnP.PowerShell', but the module could not be loaded due to the following error: [Could not load file or assembly 'Microsoft.Online.SharePoint.Client.Tenant, Version=16.1.0.0, Culture=neutral, PublicKeyToken=71e9bce111e9429c'. Could not find or load a specific file. (0x80131621)]
For more information, run 'Import-Module PnP.PowerShell'.

NOTE: The SharePoint Assembly already loaded but it's not working.

![image](https://github.com/pnp/powershell/assets/64703887/97cfa894-ed74-4e00-b659-bb302ca41ace)

**Comment by markgort86** [comment]
The cmdlet is working fine over here, are you using the most recent version of PnP.PowerShell (2.4.0)? You can download the most recent version just by running the following `Update-Module PnP.PowerShell -Scope CurrentUser`

![image](https://github.com/pnp/powershell/assets/52573368/f32edb5d-2c00-4e8d-9c0e-74977b02ba5a)
